pkg/cmd/kafka/acl/list/list.go

```diff
@@ -0,0 +1,174 @@
+				return opts.localizer.MustLocalizeError("kafka.acl.common.error.noKafkaSelected")
+			}
+
+			opts.kafkaID = cfg.Services.Kafka.ClusterID
+
+			return runList(opts)
+		},
+	}
+
+	cmd.Flags().Int32VarP(&opts.page, "page", "", cmdutil.ConvertPageValueToInt32(build.DefaultPageNumber), opts.localizer.MustLocalize("kafka.acl.list.flag.page.description"))
+	cmd.Flags().Int32VarP(&opts.size, "size", "", cmdutil.ConvertSizeValueToInt32(build.DefaultPageSize), opts.localizer.MustLocalize("kafka.acl.list.flag.size.description"))
+	cmd.Flags().StringVarP(&opts.output, "output", "o", "", opts.localizer.MustLocalize("kafka.acl.list.flag.output.description"))
+
+	return cmd
+}
+
+func runList(opts *options) (err error) {
+	conn, err := opts.Connection(connection.DefaultConfigRequireMasAuth)
+	if err != nil {
+		return err
+	}
+
+	api, kafkaInstance, err := conn.API().KafkaAdmin(opts.kafkaID)
+	if err != nil {
+		return err
+	}
+
+	req := api.AclsApi.GetAcls(opts.Context)
+
+	req.Page(float32(opts.page))
+
+	req.Size(float32(opts.size))
+
+	permissionsData, httpRes, err := req.Execute()
+	if httpRes != nil {
+		defer httpRes.Body.Close()
+	}
+
+	if err != nil {
+		if httpRes == nil {
+			return err
+		}
+
+		operationTmplPair := localize.NewEntry("Operation", "list")
+
+		switch httpRes.StatusCode {
+		case http.StatusUnauthorized:
+			return opts.localizer.MustLocalizeError("kafka.acl.common.error.unauthorized", operationTmplPair)
+		case http.StatusForbidden:
+			return opts.localizer.MustLocalizeError("kafka.acl.common.error.forbidden", operationTmplPair)
+		case http.StatusInternalServerError:
+			return opts.localizer.MustLocalizeError("kafka.acl.common.error.internalServerError")
+		case http.StatusServiceUnavailable:
+			return opts.localizer.MustLocalizeError("kafka.acl.common.error.unableToConnectToKafka", localize.NewEntry("Name", kafkaInstance.GetName()))
+		default:
+			return err
+		}
+	}
+
+	switch opts.output {
+	case dump.EmptyFormat:
+		opts.Logger.Info("")
+		permissions := permissionsData.GetItems()
+		rows := mapPermissionListResultsToTableFormat(permissions, opts.localizer)
+		dump.Table(opts.IO.Out, rows)
+	default:
+		return dump.Formatted(opts.IO.Out, opts.output, permissionsData)
+	}
+
+	return nil
+}
+
+func mapPermissionListResultsToTableFormat(permissions []kafkainstanceclient.AclBinding, localizer localize.Localizer) []permissionsRow {
+
+	rows := make([]permissionsRow, len(permissions))
+
+	for i, p := range permissions {
+
+		description := buildDescription(p.PatternType, localizer)
+		row := permissionsRow{
+			Principal:   formatPrincipal(p.GetPrincipal(), localizer),
+			Permission:  fmt.Sprintf("%s | %s", p.GetPermission(), p.GetOperation()),
+			Description: fmt.Sprintf("%s %s \"%s\"", p.GetResourceType(), description, p.GetResourceName()),
+		}
+		rows[i] = row
+	}
+	return rows
+}
+
+func formatPrincipal(principal string, localizer localize.Localizer) string {
+	s := strings.Split(principal, ":")[1]
+
+	if s == "*" {
```

Comment:
```suggestion
	if s == acl.Wildcard {
```

Something like this will be useful.